Marlin/src/lcd/tft/touch.cpp

```diff
@@ -95,6 +95,8 @@ void Touch::idle() {
       }
     #endif
 
+    SERIAL_ECHOLNPAIR("X: ", _x, ", Y: ", _y);
```

Comment:
Is this supposed to be here? It looks like a debugging statement that slipped into the PR?

Reply:
Hahaha. No. I forgot to remove after the user test it.